### PR TITLE
docs: handleAsSDL option

### PR DESCRIPTION
**Pull request by saihaj**
We get this quesiton time to time so worth documenting.

### CODE REVIEW

**Comment by n1ru4l** [comment]
Hive is a bad example because we have the sdl.graphql endpoint 🤔

File: website/src/pages/docs/config-reference/schema-field.mdx [hunk 1]
```diff
@@ -162,6 +162,26 @@ export default config;
...
+
+Handling the response as SDL will allow you to load schema from remote server that doesn't return a JSON introspection. Example use case is when using schema registry like [GraphQL Hive](https://docs.graphql-hive.com/features/high-availability-cdn#graphql-schema-sdl).
+
+```ts {7}
+import { CodegenConfig } from '@graphql-codegen/cli';
+
+const config: CodegenConfig = {
+  schema: [
+    {
+      'http://localhost:3000/graphql': {
```
Comment (ardatan) [inline website/src/pages/docs/config-reference/schema-field.mdx:175]:
```suggestion
      'http://localhost:3000/sdl': {
```
`/graphql` can be confusing.

**Comment by ardatan** [comment]
@saihaj That one has `.graphql` at the end so UrlLoader enables that option automatically.

**Comment by saihaj** [comment]
> @saihaj That one has `.graphql` at the end so UrlLoader enables that option automatically.

Not everyone would be exposing `.graphql` we can have that part of Hive docs on specifics?

**Pull request merged**